src/models/AccessRights.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * LINKURIOUS CONFIDENTIAL
+ * Copyright Linkurious SAS 2012 - 2019
+ *
+ * - Created on 2019-09-12.
+ */
+
+// TS2019-DONE
+
+import {IDataSourceParams} from './DataSource';
+
+export interface ISetAccessRightsParams extends IDataSourceParams {
+  groupId: number;
+  type: string;
```

Comment:
should be a union type `type AccesRightType = "read" | "write" | "edit" | "do" | "none"`